### ISSUE TITLE
Add comments to .localized()?

### ISSUE THREAD

**Issue by Gatada** [marmelroy/Localize-Swift#74]
Would be nice to have an optional comment added to localized() that defaults to nil.

**Comment by rlam3** [comment]
I totally agree we need this. Anybody know a workaround for this yet? @marmelroy

**Comment by calube** [comment]
fuck comments mane
